README.md

```diff
@@ -11,7 +11,8 @@ Usage
 -----
 
 Add `rspec-rerun` to `Gemfile` in the `:development` and `:test` groups.
-If you're using RSpec 3, also add `rspec-legacy_formatters`.
+
+(For RSpec 2 use version '~> 0.3.1'. )
```

Comment:
We no longer need `gem 'rspec-legacy-formatters'` anymore explicitly, should be removed from README.